<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View_form</class>
 <widget class="QWidget" name="View_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #AFEEEE;</string>
  </property>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>410</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background: #65CCB8;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>назад</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>341</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 20px;
</string>
      </property>
      <property name="text">
       <string>Расписание экзаменов на</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEditForm">
      <property name="styleSheet">
       <string notr="true">border: 0px;
color: white;
font-size: 15px;
text-decoration: underline;
</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximumDate">
       <date>
        <year>2026</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2022</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 20px;
</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>581</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_view">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background: #65CCB8;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>просмотр</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font-size: 20px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_exit</sender>
   <signal>clicked()</signal>
   <receiver>View_form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
